apps/shared/test-helpers/mine.js

```diff
@@ -1,43 +1,49 @@
-const Web3 = require("web3");
-const PrivateKeyProvider = require("truffle-privatekey-provider");
-const provider = new PrivateKeyProvider(
-    "A8A54B2D8197BC0B19BB8A084031BE71835580A01E70A45A13BABD16C9BC1563",
-    "http://localhost:8545"
-  );
-const web3 = new Web3(provider);
+/// uncomment the Below block to use on aragon devchain without truffle
```

Comment:
Did you mean to put three `///` at the beginning of this line?

Reply:
Yes because it's a metacomment about the commented code below